pandas/tests/frame/test_timezones.py

```diff
@@ -166,3 +166,19 @@ def test_constructor_data_aware_dtype_naive(self, tz_aware_fixture):
         result = DataFrame({"d": [pd.Timestamp("2019", tz=tz)]}, dtype="datetime64[ns]")
         expected = DataFrame({"d": [pd.Timestamp("2019")]})
         tm.assert_frame_equal(result, expected)
+
+    @pytest.mark.parametrize(
+        "initial",
+        ["2018-10-08 13:36:45+00:00", "2018-10-08 13:36:45+03:00"],  # Non-UTC timezone
+    )
+    @pytest.mark.parametrize("method", [DataFrame.min, DataFrame.max])
```

Comment:
Could you use the strings `['min', 'max']` here, and then below you can do `getattr(df, method)(axis=1)`

Reply:
Addressed.